ibis/expr/types/relations.py

```diff
@@ -864,6 +864,45 @@ def fillna(
                 )
         return ops.FillNa(self, replacements).to_expr()
 
+    def unpack(self, *columns: str) -> Table:
+        """Project the struct fields `columns` the table.
```

Comment:
I think this is missing a word, but I'm not sure what/where.

Reply:
I think `into the table` is what I meant to write here.